contracts/ExclusiveGeyser.sol

```diff
@@ -2,12 +2,13 @@
 pragma solidity 0.7.6;
 pragma abicoder v2;
 
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {IUniversalVault} from "./UniversalVault.sol";
 import {Geyser} from "./Geyser.sol";
 
 /// @title ExclusiveGeyser
-/// @notice A special extension of GeyserV2 which allows staking in,
-///         at most one distribution program in any given time, for a given staking token.
+/// @notice A special extension of GeyserV2 which limits staking tokens,
```

Comment:
How about
`A special extension of GeyserV2 which enforces that no staking token balance may be staked in more than one geyser at a time.`